src/iOS/toga_iOS/widgets/optioncontainer.py

```diff
@@ -0,0 +1,134 @@
+    #
+    #     result = True
+    #     return result
+
+    @objc_method
+    def tabBarController_didSelectViewController_(self,
+                                                  tbc,              # type: UITabBarController
+                                                  vc,               # type: UIViewController
+                                                  ) -> None:
+        """
+        Event callback after tab has finished transitioning to new tab.
+        """
+
+        # call the user on_select callback method (if provided)
+        if self.interface.on_select:
+            index = tbc.selectedIndex
+            option = tbc.interface.content[index]
+            self.interface.on_select(self.interface, option=option, index=index)
+
+        # required to redraw new tabs correctly (possible Toga bug?)
+        self.interface.refresh()
+
+
+class OptionContainer(Widget):
+    """
+    OptionContainer implementation for the iOS platform.
+    """
+
+    def create(self):
+        """
+        Create native widget - iOS UITabController
+        """
+
+        self.controller = TogaTabBarController.alloc().init()
+        self.controller.interface = self.interface
+
+        self.native = self.controller.view
+
+        self.controller.delegate = self.controller
+        self.native.delegate = self.controller
+
+        # a list of view controllers for each tab (see `add_content()`)
+        self.view_controllers = []
+
+        # self.native.translatesAutoresizingMaskIntoConstraints = False
+
+        # Add the layout constraints
+        self.add_constraints()
+
+    def add_content(self, label, widget):
+        """ Adds a new option to the option container.
+
+        Args:
+            label (str): The label for the option container
+            widget: The widget or widget tree that belongs to the label.
+        """
+
+        widget.viewport = iOSViewport(widget.native)
+
+        for child in widget.interface.children:
+            child._impl.container = widget
+
+        # tag = len(self.interface.content) - 1
+        tab_bar_item = UITabBarItem.alloc().init()
+        tab_bar_item.title = label
+
+        # Turn the autoresizing mask on the widget widget
+        # into constraints. This makes the widget fill the
+        # available space inside the OptionContainer.
+        widget.native.translatesAutoresizingMaskIntoConstraints = True
+
+        # create view controller for the view and associate tab bar item
+        vc = UIViewController.alloc().init()
+        vc.view = widget.native
+        vc.tabBarItem = tab_bar_item
+        vc.delegate = self.controller
+        widget.native.delegate = self.controller
+
+        # append new view controller to exisiting list of view controllers
+        self.view_controllers.append(vc)
+
+        controller = self.controller
+        # controller.viewControllers = self.view_controllers
+        # controller.setViewControllers(self.view_controllers, animated=False)
```

Comment:
A minor thing, but if there's no need to keep around this code, we should strip it out.